portable/Dockerfile

```diff
@@ -26,11 +26,8 @@ RUN apt-get update -y \
  && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC \
     apt-get install --no-install-recommends -y \
     wget debhelper gnupg devscripts build-essential mmv lsb-release zip \
-    libssl*.* liblttng-ust* \
```

Comment:
And I realize this conflicts with my previous comments about wanting them all on new lines for readability, so that's what I'd suggest here, move them to separate lines here rather than removing the separate lines above.